packages/contracts/hardhat.config.ts

```diff
@@ -148,28 +148,103 @@ const config: HardhatUserConfig = {
     apiKey: {
       mainnet: process.env.ETHERSCAN_API_KEY || '',
       sepolia: process.env.ETHERSCAN_API_KEY || '',
+      holesky: process.env.ETHERSCAN_API_KEY || '',
       polygon: process.env.POLYGONSCAN_API_KEY || '',
-      base: process.env.BASESCAN_API_KEY || '',
+      polygonMumbai: process.env.POLYGONSCAN_API_KEY || '',
+      baseMainnet: process.env.BASESCAN_API_KEY || '',
+      baseGoerli: process.env.BASESCAN_API_KEY || '',
+      baseSepolia: process.env.BASESCAN_API_KEY || '',
       arbitrumOne: process.env.ARBISCAN_API_KEY || '',
+      arbitrumSepolia: process.env.ARBISCAN_API_KEY || '',
+      modeTestnet: 'modeTestnet',
+      modeMainnet: 'modeMainnet',
+      lineaSepolia: process.env.LINEASCAN_API_KEY || '',
+      linea: process.env.LINEASCAN_API_KEY || '',
       peaq: '1',
     },
     customChains: [
       {
-        network: 'sepolia',
-        chainId: 11155111,
+        network: 'polygon',
```

Comment:
It looks like there are more networks defined on `apiKey` than actually implemented here. Is this intended? (polygonMumbai, etc)